docs/content/installation/upgrade_manual_openshift.md

```diff
@@ -34,4 +34,8 @@ When using the Red Hat® OpenShift® Container Platform, the CSI (Container Stor
 
 12. Check the **Overview** tab and that the **Controller Image Tab** and **Node Image Tag** are showing the most up-to-date version of the driver and the **Status** is _Running_.
 
+13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```

Comment:
```suggestion
13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```

Reply:
updated
note - "IBMBlockCSI_hostdefiner" is just a placeholder.
waiting on @matancarmeli7  for final name